### PR TITLE
Added Docker container support

### PR DESCRIPTION
**Pull request by jgrim**
Containerized overseer.
Called the example the eye. 
Containerized that too.

### CODE REVIEW

**Comment by jgrim** [comment]
I haven't tried this yet...

I should probably update the README.md with environmental variable instructions, etc.

I wanted you to see it is in the works.

File: overseer/__init__.py [hunk 1]
```diff
@@ -8,7 +9,7 @@
 from overseer.apis import apiv1
 from overseer.argparser import args
 from overseer.consts import OVERSEER_VERSION
-
+logger.add(sys.stdout)
```
Comment (db0) [inline overseer/__init__.py:12]: We already have stdout output. Please check logger.py

**Comment by db0** [comment]
The code had received a major update, and now has a postgresql involved

**Comment by db0** [comment]
@jgrim are you planning to fix this or should I close it?

**Comment by jgrim** [comment]
I'll close it. I can reengage if there becomes a need. Seems like there won't be.

**Pull request closed without merging**